src/app/header/header.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-default">
```

Comment:
Create shared module https://angular.io/guide/sharing-ngmodules (find more complex example via google). Move header to shared module + create footer component (also in shared module)

Reply:
done